frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.tsx

```diff
@@ -30,7 +30,7 @@ export const StudioRecommendedNextAction = ({
 }: StudioRecommendedNextActionProps): React.ReactElement => {
   const formName = useId();
   return (
-    <form name={formName} onSubmit={onSave}>
+    <form name={formName} onSubmit={onSave} data-testid='recommendedNextActionCard'>
```

Comment:
We have a nice list of dataids used here: https://github.com/Altinn/altinn-studio/blob/fc5f892a26ca5bf49189d327abcfbd7210671844/frontend/testing/testids.js#L4 
Could we add this id here?